### DISCUSSION TITLE
Change CameraSensor used by Budibase

### DISCUSSION THREAD

**Discussion by Tim0710h** [Budibase/budibase#14167] [Help] [answered]
Hi, I've created an app with Budibase, where you can scan a QR-Code to get some information about a product. But the Problem is, that by default the ultrawide camera sensor from my smartphone is used, which would require verry big QR-Codes, because it cant focus to such a distance where the QR-Code wont get to small to be still readable. I'm using an Samsung S20+, does anyone has an idea how to solve this problem?
Thanks in advance!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @TimPlay38,

I've create a feature request [here](https://github.com/Budibase/budibase/issues/14171) to get this functionality added to the product.
